go.mod

```diff
@@ -6,6 +6,8 @@ replace cloud.google.com/go => cloud.google.com/go v0.102.1
 
 replace github.com/vmware-tanzu/tanzu-cli/test/e2e/framework => ./test/e2e/framework
 
+replace github.com/vmware-tanzu/tanzu-plugin-runtime => ../tanzu-plugin-runtime
```

Comment:
nit: It might be better to point this to your fork repository from where the PR https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/126 is created so that we can verify that CI run is successful until that PR gets in.

Update:
```
replace github.com/vmware-tanzu/tanzu-plugin-runtime => github.com/vuil/tanzu-plugin-runtime eula-accepted-versions
```
And run `make gomod` to update the `go.mod` and `go.sum` files.

Reply:
yep, will keep that in mind. Given that 126 merged, I have updated the go mod